pkg/lint/lint.go

```diff
@@ -25,19 +25,19 @@ import (
 )
 
 // All runs all of the available linters on the given base directory.
-func All(basedir string, values map[string]interface{}, namespace string, _ bool) support.Linter {
-	return AllWithKubeVersion(basedir, values, namespace, nil)
+func All(basedir string, values map[string]interface{}, namespace string, quiet bool) support.Linter {
+	return AllWithKubeVersion(basedir, values, namespace, nil, quiet)
 }
 
 // AllWithKubeVersion runs all the available linters on the given base directory, allowing to specify the kubernetes version.
-func AllWithKubeVersion(basedir string, values map[string]interface{}, namespace string, kubeVersion *chartutil.KubeVersion) support.Linter {
+func AllWithKubeVersion(basedir string, values map[string]interface{}, namespace string, kubeVersion *chartutil.KubeVersion, quiet bool) support.Linter {
```

Comment:
This is a public function and the arguments are changing. This can only happen when the major version of Helm is incremented for [backwards compatability](https://github.com/helm/community/blob/main/hips/hip-0004.md). Those who use Helm as an SDK would have a broken experience, otherwise.

We normally handle this by having two functions. One with the old signature and behavior and one with the new name and the different signature. Often, the old one will call the new one.

Can you please ensure backwards compatibility.

Reply:
@mattfarina  Thanks for reaching out,
I've created a new public function and updated it's reference in 8bdcd9d.
I also add a simple test `TestLint_ChartWithInfo` that checks there are no `INFO` messages.

I forgot to sign-off some commits, therefore the force-push, sorry about it!